### ISSUE TITLE
Coal Boilers still explode

### ISSUE THREAD

**Issue by ZeekDaGeek** [GregTechCE/GregTech#762]
Small Steam Coal Boilers (SSCB) seem to explode when they shouldn't.

1. Let a SSCB heat up to max.
![N2HBu](https://user-images.githubusercontent.com/1709288/56457776-beb17800-633c-11e9-983d-4ff921e24174.png)
2. Let the SSCB run out of water.
3. Let the SSCB cool down completely.
4. Try adding water to the cooled boiler and second it creates it's first 30 MB of steam it will explode.

**Comment by Archengius** [comment]
either it didn't cool down to below 100C in your case or you are running outdated version. cannot reproduce.

**Comment by jnaset** [comment]
I had experienced this (on a previous version of GTCE included in the Gregblock 2.0 pack) except with a solar boiler.  The distinctive thing about the explosion was it didn't occur when the water was added, it waited until after the boiler heated up again before blowing.  I thought it was weird and it only happened twice, but if I had added water with the boiler over 100C shouldn't it have exploded right then and there?

Anyway, if it's no longer reproducible that's great!  I'll keep my eyes open for this one though.
